tests/AbstractIndexingTestCase.php

```diff
@@ -7,30 +7,45 @@ class AbstractIndexingTestCase extends TestCase
     /** @var Algolia_Algoliasearch_Helper_Algoliahelper */
     protected $algoliaHelper;
 
+    protected $indexPrefix;
+
+    /**
+     * @var array Default value of general store settings (not related to algolia's extention)
+     */
+    protected $defaultConfig = array(
+        'cataloginventory/options/show_out_of_stock' => '0',
+    );
+
+    const DEFAULT_PRODUCT_COUNT = 86;
```

Comment:
I'd move the constant at least to ProductIndexingTest.php as it's relevant only to products and not other children of the class.